sections/exchange/MobileSwap/RatioSelect.tsx

```diff
@@ -26,7 +26,7 @@ const RatioSelect: FC = () => {
 				<RatioButton
 					key={`ratio-${v}`}
 					$selected={ratio === v}
-					onClick={() => onRatioChange(v)}
+					onClick={onRatioChange(v)}
```

Comment:
Not sure I agree with this change as it's a pattern not used anywhere else throughout the codebase so presumed it was a bug until I seen you added the call step above. Would prefer if we reverted to remain consistent.

Reply:
Changed.